AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusElementHighlight.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.accessibilityinsightsforandroidservice;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.View;
+import android.view.accessibility.AccessibilityNodeInfo;
+import java.util.HashMap;
+
+public class FocusElementHighlight {
+  private static final String TAG = "FocusElementHighlight";
```

Comment:
IIRC, we only use `TAG` when we're logging. If no logging is being added, can it go away? (applies to 3 classes)

Reply:
I've removed the TAGs.